tests/notebooks/e2e-wine/e2e-wine-kfp-mlflow-seldon.ipynb

```diff
@@ -0,0 +1,1233 @@
+    "    secret_key=os.environ[\"AWS_SECRET_ACCESS_KEY\"],\n",
+    "    secure=False,\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "id": "f86b38b9-f7df-4f21-9641-fbe15c952010",
+   "metadata": {
+    "execution": {
+     "iopub.execute_input": "2023-08-09T14:52:37.754470Z",
+     "iopub.status.busy": "2023-08-09T14:52:37.753652Z",
+     "iopub.status.idle": "2023-08-09T14:52:37.814713Z",
+     "shell.execute_reply": "2023-08-09T14:52:37.765115Z"
+    }
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Bucket mlflow already exists!\n"
+     ]
+    }
+   ],
+   "source": [
+    "try:\n",
+    "    mc.make_bucket(MINIO_BUCKET)\n",
+    "except BucketAlreadyOwnedByYou:\n",
+    "    print(f\"Bucket {MINIO_BUCKET} already exists!\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "85e6967b",
+   "metadata": {},
+   "source": [
+    "## Download Data"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "id": "8b5e4a40-8acc-45d0-9b13-86a2b5f27ce0",
+   "metadata": {
+    "execution": {
+     "iopub.execute_input": "2023-08-09T14:52:37.822193Z",
+     "iopub.status.busy": "2023-08-09T14:52:37.820945Z",
+     "iopub.status.idle": "2023-08-09T14:52:37.827564Z",
+     "shell.execute_reply": "2023-08-09T14:52:37.826204Z"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "DATA_URL = \"https://raw.githubusercontent.com/canonical/kubeflow-examples/main/e2e-wine-kfp-mlflow/winequality-red.csv\"\n",
+    "DATA_FILE = \"winequality-red.csv\""
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "id": "6442ab5f-2c87-49e4-88d3-4a88b3ec1eb9",
+   "metadata": {
+    "execution": {
+     "iopub.execute_input": "2023-08-09T14:52:37.833566Z",
+     "iopub.status.busy": "2023-08-09T14:52:37.831996Z",
+     "iopub.status.idle": "2023-08-09T14:52:38.058836Z",
+     "shell.execute_reply": "2023-08-09T14:52:38.057568Z"
+    }
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "File 'winequality-red.csv' downloaded successfully.\n"
+     ]
+    }
+   ],
+   "source": [
+    "# local development\n",
+    "request.urlretrieve(DATA_URL, DATA_FILE)\n",
+    "\n",
+    "print(f\"File '{DATA_FILE}' downloaded successfully.\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "id": "ebc1a73b",
+   "metadata": {
+    "execution": {
+     "iopub.execute_input": "2023-08-09T14:52:38.064218Z",
+     "iopub.status.busy": "2023-08-09T14:52:38.063390Z",
+     "iopub.status.idle": "2023-08-09T14:52:38.296084Z",
+     "shell.execute_reply": "2023-08-09T14:52:38.291832Z"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "# workflow component\n",
+    "web_downloader_op = kfp.components.load_component_from_url(\n",
+    "    \"https://raw.githubusercontent.com/kubeflow/pipelines/master/components/contrib/web/Download/component.yaml\"\n",
```

Comment:
Should we pin this to a release tag? Maybe 2.0.0 https://github.com/kubeflow/pipelines/blob/2.0.0/components/contrib/web/Download/component.yaml to avoid potential 404s?

Reply:
Good suggestion!

Reply:
Pinning this to 1.8.22, since I'm keeping the same version of the SDK.